components/org/equinet/credential-details.jsx

```diff
@@ -1,17 +1,17 @@
 import React, { useState } from 'react';
-import { toast } from 'sonner';
+import { useRevoke } from 'hooks/useRevoke';
 import { Loader2, Check, ShieldX } from 'lucide-react';
 import { Button } from 'components/ui/button';
 
 /**
  * @description Equinet Credentials comp
- * @param {*} isLoading revoke/issuing credential loading status
- * @param {*} setIsLoading trigger loading modal when revoke/issuing credential
+ * @param {*} loadingRevokation revoke/issuing credential loading status
+ * @param {*} setloadingRevokation trigger loading modal when revoke/issuing credential
```

Comment:
`loadingRevokation` and `setLoadingRevokation` are coming from `useRevoke` hook so they should not be listed as `@params`